test/error.test.js

```diff
@@ -136,30 +142,44 @@ describe('seneca-error', function () {
     test_action(si, ctxt)
   }
 
+  // REMOVE after Seneca 3.x
```

Comment:
We still test older versions in our test matrix, I suggest we have .legacy.test for this stuff.